core/types/src/quote.rs

```diff
@@ -125,67 +125,67 @@ impl_newtype! {
 pub trait BaseQuote {
     /// Provides access to the [`RawQuote`] to perform the common lookup
     /// operations on the basic quote type.
-    fn raw_quote(&self) -> &RawQuote;
+    fn _raw_quote(&self) -> &RawQuote;
```

Comment:
I chose to underscore for now.
Another alternative is to ensure this is exported publicly in `lib.rs`, however since this is related to the older EPID quote, I'm not sure adding to the public API for something that we know will be going away is a good choise.